spec/author_spec.rb

```diff
@@ -0,0 +1,21 @@
+require_relative 'helper_spec'
+
+describe Author do
+  before :each do
+    @zabuza = Author.new('Deidara', 'Danzo')
+  end
+
+  context 'Test for Author class' do
+    it 'should be instance of author' do
+      expect(@zabuza).to be_an_instance_of Author
+    end
+
+    it 'should return first-name' do
+      expect(@zabuza.first_name).to eq('Deidara')
+    end
+
+    it 'should return last-name' do
+      expect(@zabuza.last_name).to eq('Danzo')
+    end
+  end
+end
```

Comment:
- [ ] Great job so far 👏🏻 👏🏻 However, according to the project requirements, All public methods in all classes should have unit tests, Kindly make sure the `add_item` class classroom method has (at least) an 'it' block 👇🏻 check your author.rb file

```ruby
   def add_item(item)
      @items.push(item)
      item.add_author(self)
   end
```